docker/package/tezos_voting_wizard.py

```diff
@@ -142,23 +169,97 @@ def check_baking_service(self):
         net = self.config["network"]
         try:
             proc_call(f"systemctl is-active --quiet tezos-baking-{net}.service")
+            return True
         except:
             print(f"Looks like the tezos-baking-{net} service isn't running.")
-            print("Please start the service or set up baking.")
-            if self.config["network_mode"] == "mainnet":
-                print("You can do this by running:")
-                print("tezos-setup-wizard")
-            sys.exit(1)
+            print("If this is a mistake, and you should have a baking instance")
+            print("running on mainnet on this machine, please check if it is set up.")
+            print("If it isn't, you can use Tezos Setup Wizard to set it up:")
+            print("tezos-setup-wizard")
+            print()
+
+            return False
 
     def check_data_correctness(self):
         print("Baker data detected is as follows:")
         print(f"Data directory: {self.config['client_data_dir']}")
         print(f"Node RPC address: {self.config['node_rpc_addr']}")
-        print(f"Baker alias: {self.config['baker_alias']}")
-        if not yes_or_no("Does this look correct? (Y/n)", "yes"):
-            print("Try setting up baking again by running:")
-            print("tezos-setup-wizard")
-            sys.exit(1)
+        print(f"Voter key: {self.config['baker_key_value']}")
+        return yes_or_no("Does this look correct? (Y/n) ", "yes")
+
+    def search_client_config(self, field, default):
+        config_filepath = os.path.join(self.config["client_data_dir"], "config")
+        if not os.path.isfile(config_filepath):
+            return default
+        else:
+            return search_json_with_default(config_filepath, field, default)
+
+    def collect_baking_info(self):
+        if self.check_baking_service():
+            self.fill_baking_config()
+
+            value, _ = get_key_address(
+                self.config["tezos_client_options"], self.config["baker_alias"]
+            )
+            self.config["baker_key_value"] = value
+
+            collected = self.check_data_correctness()
+        else:
+
+            network_dir = "/var/lib/tezos/client-" + self.config["network"]
+
+            proc_call(f"sudo -u tezos mkdir -p {network_dir}")
+
+            print("With no tezos-baking.service running, this wizard will use")
+            print(f"the default directory for this network: {network_dir}")
+
+            self.config["client_data_dir"] = network_dir
+
+            self.config["node_rpc_addr"] = self.search_client_config("endpoint", None)
+            if self.config["node_rpc_addr"] is None:
+                self.query_step(get_node_rpc_addr_query())
+                self.config["tezos_client_options"] = self.get_tezos_client_options()
+                proc_call(
+                    f"sudo -u tezos {suppress_warning_text} tezos-client "
```

Comment:
For some reason, when the node endpoint is already present in the client config and another endpoint was used during voting, an endpoint in the config isn't updated. Is that expected?
```
Baker data detected is as follows:
Data directory: /var/lib/tezos/client-mainnet
Node RPC address: http://localhost:8734
Voter key: ...
Does this look correct? (Y/n) n

Provide the node's RPC address.
> http://localhost:8735
...
Thank you for voting!
$ tezos-voting-wizard 
Tezos Voting Wizard
...
With no tezos-baking.service running, this wizard will use
the default directory for this network: /var/lib/tezos/client-mainnet
Baker data detected is as follows:
Data directory: /var/lib/tezos/client-mainnet
Node RPC address: http://localhost:8734
Voter key: ...
```

Reply:
No, thanks for catching that. Made a fix